### PR TITLE
Added code for Intersection of two linked lists

### PR DESCRIPTION
**Pull request by SmritiOjha23**
Fixes issue #85

### CODE REVIEW

**Review by nimisha-yadav — CHANGES_REQUESTED** [review]
Hey @SmritiOjha23, please explain your algorithm at the beginning of the code.

**Comment by SmritiOjha23** [comment]
@nimisha-yadav I have updated the file with the required changes. Please check.

**Review by nimisha-yadav — APPROVED** [review]
LGTM

File: Linked list/Intersection of Two Linked Lists.java [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+	{
+		//Function to add nodes to the respective linked lists
+		ListNode node=new ListNode(0);
+		node.val=data;
+		curr.next=node;
+		curr=curr.next;
+	}
+	
+	
+	public static void intersect(ListNode last1,ListNode last2, ListNode commonHead)
```
Comment (nimisha-yadav) [inline Linked list/Intersection of Two Linked Lists.java:60]: error: cannot find symbol [in __Driver__.java]

**Review by nimisha-yadav — CHANGES_REQUESTED** [review]

**Review by nimisha-yadav — CHANGES_REQUESTED** [review]
Error: Exception in thread "main"

**Comment by SmritiOjha23** [comment]
@nimisha-yadav The code runs without any errors on Java IDE's like Net beans and Eclipse. However, if a person runs this on online compilers like Codechef, one has to do a small amendment in the definition of the main function else it will throw a runtime error.

**Comment by SmritiOjha23** [comment]
@nimisha-yadav please check. I hope it works fine now.

**Review by nimisha-yadav — APPROVED** [review]
LGTM!

**Review by Ananyaas — APPROVED** [review]

**Review by sakshigoel19 — APPROVED** [review]

**Review by v-vanisha — APPROVED** [review]

**Review by Vanshikagarg17 — APPROVED** [review]

**Pull request merged**